`bazel sync` error in docker

Running `bazel sync` inside docker creates the following error:
```
INFO: Call stack for the definition of repository 'local_config_python' which is a python_configure (rule definition at /root/.cache/bazel/_bazel_root/b6bc0828d5b43c7085b650e0698bf388/external/co
m_github_grpc_grpc/third_party/py/python_configure.bzl:294:20):                                                                                                                                    
 - /root/.cache/bazel/_bazel_root/b6bc0828d5b43c7085b650e0698bf388/external/com_github_grpc_grpc/bazel/grpc_python_deps.bzl:57:5                                                                   
 - /root/.cache/bazel/_bazel_root/b6bc0828d5b43c7085b650e0698bf388/external/com_github_grpc_grpc/bazel/grpc_deps.bzl:233:5                                                                         
 - /opt/my-project/WORKSPACE:196:1                                                                                                                                                                 
ERROR: An error occurred during the fetch of repository 'local_config_python':                                                                                                                     
   Traceback (most recent call last):                                                                                                                                                              
        File "/root/.cache/bazel/_bazel_root/b6bc0828d5b43c7085b650e0698bf388/external/com_github_grpc_grpc/third_party/py/python_configure.bzl", line 281                                         
                _create_single_version_package(repository_ctx, "_python2", _PYTHON2..., <2 more arguments>)                                                                                        
        File "/root/.cache/bazel/_bazel_root/b6bc0828d5b43c7085b650e0698bf388/external/com_github_grpc_grpc/third_party/py/python_configure.bzl", line 255, in _create_single_version_package      
                _symlink_genrule_for_dir(repository_ctx, python_include, "{}_...), ...))                                                                                                           
        File "/root/.cache/bazel/_bazel_root/b6bc0828d5b43c7085b650e0698bf388/external/com_github_grpc_grpc/third_party/py/python_configure.bzl", line 119, in _symlink_genrule_for_dir            
                "\n".join(sorted(_read_dir(repository_ctx,...()))                                                                                                                                  
        File "/root/.cache/bazel/_bazel_root/b6bc0828d5b43c7085b650e0698bf388/external/com_github_grpc_grpc/third_party/py/python_configure.bzl", line 120, in "\n".join                           
                sorted(_read_dir(repository_ctx, src_di...())                                                                                                                                      
        File "/root/.cache/bazel/_bazel_root/b6bc0828d5b43c7085b650e0698bf388/external/com_github_grpc_grpc/third_party/py/python_configure.bzl", line 120, in sorted                              
                _read_dir(repository_ctx, src_dir)                                                                                                                                                 
        File "/root/.cache/bazel/_bazel_root/b6bc0828d5b43c7085b650e0698bf388/external/com_github_grpc_grpc/third_party/py/python_configure.bzl", line 81, in _read_dir                            
                _execute(repository_ctx, ["find", src_dir, ..."], ...)                                                                                                                             
        File "/root/.cache/bazel/_bazel_root/b6bc0828d5b43c7085b650e0698bf388/external/com_github_grpc_grpc/third_party/py/python_configure.bzl", line 56, in _execute                             
                _fail("\n".join([error_msg.strip() if ... ""]))                                                                                                                                    
        File "/root/.cache/bazel/_bazel_root/b6bc0828d5b43c7085b650e0698bf388/external/com_github_grpc_grpc/third_party/py/python_configure.bzl", line 28, in _fail                                
                fail(("%sPython Configuration Error:%...)))                                                                                                                                        
Python Configuration Error: Repository command failed                                                                                                                                              
find: '/usr/include/python2.7': No such file or directory                                                                                                                                          
                                                                                                                                                                                                   
ERROR: Traceback (most recent call last):                                                                                                                                                          
        File "/root/.cache/bazel/_bazel_root/b6bc0828d5b43c7085b650e0698bf388/external/com_github_grpc_grpc/third_party/py/python_configure.bzl", line 281                                         
                _create_single_version_package(repository_ctx, "_python2", _PYTHON2..., <2 more arguments>)
        File "/root/.cache/bazel/_bazel_root/b6bc0828d5b43c7085b650e0698bf388/external/com_github_grpc_grpc/third_party/py/python_configure.bzl", line 255, in _create_single_version_package
                _symlink_genrule_for_dir(repository_ctx, python_include, "{}_...), ...))
        File "/root/.cache/bazel/_bazel_root/b6bc0828d5b43c7085b650e0698bf388/external/com_github_grpc_grpc/third_party/py/python_configure.bzl", line 119, in _symlink_genrule_for_dir
                "\n".join(sorted(_read_dir(repository_ctx,...()))
        File "/root/.cache/bazel/_bazel_root/b6bc0828d5b43c7085b650e0698bf388/external/com_github_grpc_grpc/third_party/py/python_configure.bzl", line 120, in "\n".join
                sorted(_read_dir(repository_ctx, src_di...())
        File "/root/.cache/bazel/_bazel_root/b6bc0828d5b43c7085b650e0698bf388/external/com_github_grpc_grpc/third_party/py/python_configure.bzl", line 120, in sorted
                _read_dir(repository_ctx, src_dir)
        File "/root/.cache/bazel/_bazel_root/b6bc0828d5b43c7085b650e0698bf388/external/com_github_grpc_grpc/third_party/py/python_configure.bzl", line 81, in _read_dir
                _execute(repository_ctx, ["find", src_dir, ..."], ...)
        File "/root/.cache/bazel/_bazel_root/b6bc0828d5b43c7085b650e0698bf388/external/com_github_grpc_grpc/third_party/py/python_configure.bzl", line 56, in _execute
                _fail("\n".join([error_msg.strip() if ... ""]))
        File "/root/.cache/bazel/_bazel_root/b6bc0828d5b43c7085b650e0698bf388/external/com_github_grpc_grpc/third_party/py/python_configure.bzl", line 28, in _fail
                fail(("%sPython Configuration Error:%...)))
Python Configuration Error: Repository command failed
find: '/usr/include/python2.7': No such file or directory
```

New Asylo version shows the following error:
```shell
ERROR: /opt/my-project/oak/server/dev/BUILD:46:1: C++ compilation of rule '//oak/server/dev:oak' failed (Exit 1) clang.sh failed: error executing command toolchain/clang.sh -MD -MF bazel-out/k8-f
astbuild/bin/oak/server/dev/_objs/oak/dev_oak_main.d '-frandom-seed=bazel-out/k8-fastbuild/bin/oak/server/dev/_objs/oak/dev_oak_main.o' ... (remaining 92 argument(s) skipped)

Use --sandbox_debug to see verbose messages from the sandbox
In file included from oak/server/dev/dev_oak_main.cc:28:
In file included from oak/server/dev/dev_oak_manager.h:25:
In file included from ./oak/server/oak_runtime.h:32:
In file included from ./oak/server/storage/storage_node.h:24:
In file included from ./oak/server/storage/storage_processor.h:22:
external/com_google_asylo/asylo/crypto/aes_gcm_siv.h:76:1: error: attribute 'deprecated' is ignored, place it after "class" to apply attribute to type declaration [-Werror,-Wignored-attributes]
ABSL_DEPRECATED("AesGcmSivCryptor will be removed in a future release")
^
external/com_google_absl/absl/base/macros.h:149:49: note: expanded from macro 'ABSL_DEPRECATED'
#define ABSL_DEPRECATED(message) __attribute__((deprecated(message)))
                                                ^
1 error generated.
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM gcr.io/asylo-framework/asylo:buildenv-v0.5.0
+FROM gcr.io/asylo-framework/asylo:buildenv-v0.5.1
 
 RUN apt-get --yes update && apt-get install --yes \
   clang-format \

--- a/Dockerfile
+++ b/Dockerfile
@@ -6,7 +6,7 @@ RUN apt-get --yes update && apt-get install --yes \
   curl \
   git \
   libncurses5 \
-  python3-pip \
+  python3-six \
   shellcheck \
   xml2
 

--- a/WORKSPACE
+++ b/WORKSPACE
@@ -66,11 +66,11 @@ http_archive(
 # Asylo Framework.
 http_archive(
     name = "com_google_asylo",
-    sha256 = "7f53d61d91a8d6963a5665088b02129d643a4cfbcab883f1076b71bcbe07184d",
-    strip_prefix = "asylo-088ea3490dd4579655bd5b65b0e31fe18de7f6dd",
+    sha256 = "bc500dab6673fc6d33f7f1d6df7ba406c46992985d4c301092e6bd58732d2646",
+    strip_prefix = "asylo-6f809ed623bc33900b420b5c513829257a40f548",
     urls = [
-        # Head commit on 2019-11-21.
-        "https://github.com/google/asylo/archive/088ea3490dd4579655bd5b65b0e31fe18de7f6dd.tar.gz",
+        # Head commit on 2020-01-07.
+        "https://github.com/google/asylo/archive/6f809ed623bc33900b420b5c513829257a40f548.tar.gz",
     ],
 )
 

--- a/WORKSPACE
+++ b/WORKSPACE
@@ -66,11 +66,11 @@ http_archive(
 # Asylo Framework.
 http_archive(
     name = "com_google_asylo",
-    sha256 = "1ac2b321969bb958dcd17e45a5027395d5e8f25f210fb47334458f8f27340c19",
-    strip_prefix = "asylo-f3b0e5099d460540053b04dfe7ae95f805dda1c7",
+    sha256 = "172b8a690c36a41280813fbb124bc5a3903428fe794107070c58814821bf5ac0",
+    strip_prefix = "asylo-cf74bd57d504a6f9ef52a46fdce6230e9d44f542",
     urls = [
-        # Head commit on 2020-01-16.
-        "https://github.com/google/asylo/archive/f3b0e5099d460540053b04dfe7ae95f805dda1c7.tar.gz",
+        # Head commit on 2020-01-15.
+        "https://github.com/google/asylo/archive/cf74bd57d504a6f9ef52a46fdce6230e9d44f542.tar.gz",
     ],
 )
 

--- a/WORKSPACE
+++ b/WORKSPACE
@@ -66,11 +66,11 @@ http_archive(
 # Asylo Framework.
 http_archive(
     name = "com_google_asylo",
-    sha256 = "bc500dab6673fc6d33f7f1d6df7ba406c46992985d4c301092e6bd58732d2646",
-    strip_prefix = "asylo-6f809ed623bc33900b420b5c513829257a40f548",
+    sha256 = "1ac2b321969bb958dcd17e45a5027395d5e8f25f210fb47334458f8f27340c19",
+    strip_prefix = "asylo-f3b0e5099d460540053b04dfe7ae95f805dda1c7",
     urls = [
-        # Head commit on 2020-01-07.
-        "https://github.com/google/asylo/archive/6f809ed623bc33900b420b5c513829257a40f548.tar.gz",
+        # Head commit on 2020-01-16.
+        "https://github.com/google/asylo/archive/f3b0e5099d460540053b04dfe7ae95f805dda1c7.tar.gz",
     ],
 )
 

--- a/oak/server/asylo/asylo_oak_loader.cc
+++ b/oak/server/asylo/asylo_oak_loader.cc
@@ -66,7 +66,7 @@ void AsyloOakLoader::InitializeEnclaveManager() {
   enclave_manager_ = manager_result.ValueOrDie();
   LOG(INFO) << "Enclave manager initialized";
   LOG(INFO) << "Loading enclave code from " << enclave_path_;
-  enclave_loader_ = absl::make_unique<asylo::SimLoader>(enclave_path_,
+  enclave_loader_ = absl::make_unique<asylo::SgxLoader>(enclave_path_,
                                                         /*debug=*/true);
 }
 

--- a/oak/server/asylo/asylo_oak_loader.h
+++ b/oak/server/asylo/asylo_oak_loader.h
@@ -51,7 +51,7 @@ class AsyloOakLoader {
 
   asylo::EnclaveManager* enclave_manager_;
   // TODO: Use non-simulated loader.
-  std::unique_ptr<asylo::SimLoader> enclave_loader_;
+  std::unique_ptr<asylo::SgxLoader> enclave_loader_;
   std::string enclave_path_;
 
   uint64_t application_id_;

